### DISCUSSION TITLE
Ionization by electric discharge

### DISCUSSION THREAD

**Discussion by zenineasa** [BLAST-WarpX/warpx#4549] [Q&A] [answered]
Hello everyone,

Thank you for reading my question.

I am looking to model and simulate ionization of hydrogen using a high volate DC source connected to two parallel plates. I am thinking about having some hydrogen particles generated and sent from one side, which enters a uniform electric field.

![image](https://github.com/ECP-WarpX/WarpX/assets/11295184/6b487e37-5c5d-4ccd-bb95-e5f24507ff23)

Could you let me know if this is an approach that would work and can be done with WarpX? Also, is there a better approach to perform the same?

Regards,
Zenin

**Comment by RemiLehe** [comment, marked as answer]
@zenineasa 
To make sure I understand: is the plan to have the particle injector emit neutral hydrogen, and then use our field ionization module to ionize these particles? 
If yes, then I think that this could be done in WarpX. You could look at this example to see how to setup field ionization:
https://github.com/ECP-WarpX/WarpX/tree/development/Examples/Tests/ionization

**Comment by zenineasa** [comment]
@RemiLehe ,
Thank you for your response. The plan is indeed to emit neutral hydrogen. But the idea is to use DC electric field to perform ionization. I see that the example you suggested use a laser, which, according my current understanding, has AC electric field that causes the ionization. I read the following wikipedia article, which suggests that there is some difference in ionization using AC and DC electric fields.
https://en.wikipedia.org/wiki/Tunnel_ionization

Can we work with DC electric field in WarpX to simulate ionization of hydrogen?

**Comment by RemiLehe** [comment]
Thanks for following up.

WarpX's field ionization module uses the ADK ionization model, which I think should work both for DC field as well as AC field -- provided that the frequency of the AC field (a laser in the above example) is much smaller than the frequency associated with the ionization energy (`E_ionization / h`). This is because, in my understanding, ADK theory assumes that the field seen by the atom varies slowly in time.

**Comment by zenineasa** [comment]
Thank you for the information!

**Comment by ax3l** [comment]
Just adding:
Although this is initially field ionization, be careful if a true discharge-driven ionization happens (from strong currents) that you might want to estimate of collisional ionization becomes relevant in this scenario, too.
